common/app/templates/headerInlineJS/shouldEnhance.scala.js

```diff
@@ -31,7 +32,8 @@
     };
 
     window.shouldEnhance = !personPrefersCore() && !isOlderDevice();
-})(navigator, window)
+    window.shouldEnhance || console && console.info && console.info("THIS IS CORE");
```

Comment:
If `window.console` is undefined, referencing `console` alone will lead to a `ReferenceError`, so you might want to change this to `window.console && window.console.info && …`

We do this enough in pre-module-loader JS to warrant a simple abstraction for it (i.e. `hasConsole` fn)